aicsimageio/utils/ome_utils.py

```diff
@@ -0,0 +1,145 @@
+"""
```

Comment:
nitpick: can we move this entire file to the metadata module: https://github.com/AllenCellModeling/aicsimageio/tree/main/aicsimageio/metadata

or the content of this file to `metadata/utils`? Was there a reason for _not_ doing that?

Reply:
nope, will do